spec/presenters/summary_presenter_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe SummaryPresenter, type: :model do
```

Comment:
is this actually a `:model` test or a unit test of something else (a presenter)?